_tutorials/new-component/create-a-new-component-part-1.md

```diff
@@ -0,0 +1,237 @@
+
+## Package Management
+
+Origami components use [Bower](https://bower.io/), a package manager like [NPM](https://www.npmjs.com/), to manage dependencies. The `bower.json` file lists the components dependencies, and points to the main Sass and JavaScript files of the component. One benefit of using Bower is it ensures a flat dependency tree, so two versions of the same component are not install at once.
+
+Although Origami components use Bower to install dependencies, developer dependencies may be installed using the [NPM](https://www.npmjs.com/) package manager, as seen in `package.json`. Rules for package management are defined in the [package management section of the specification](/spec/v1/components/#package-management).
+
+Although Origami components are authored using Bower, components are published to NPM so projects which use Origami may choose to use NPM over Bower ([but we still recommended Bower for now](/docs/tutorials/npm/)).
+
+Don't worry if this sounds a little confusing at the moment. Later in this tutorial we will install dependencies using Bower for our component to use. We will also see how Origami components are automatically published to NPM.
+
+## Start Developing
+
+Now we have a basic component to work from we can start developing!
+
+First we need to install any component dependencies by running the Origami Build Tools `install` command:
+
+<pre><code class="o-syntax-highlight--bash">obt install</code></pre>
+
+Then we can use the `develop` (`dev`) command to start working on the component. The `dev` command creates a server for us to preview our component in the browser. And whenever we make any change to the source code the component will be rebuilt, which we will be able to see by refreshing our browser:
+
+<pre><code class="o-syntax-highlight--bash">obt dev</code></pre>
+
+Opening the link output by the develop command, for example `localhost:8999`, shows us the component demos (html files) and demo assets (built js and css files) in the browser.
+
+<figure>
+	<img alt="" src="/assets/images/tutorial-new-component/example-dev-output.png" />
+	<figcaption class="o-typography-caption">
+        Running the dev command builds our demos. We'll discuss demos more shortly.
+	</figcaption>
+</figure>
+
+<figure>
+	<img alt="" src="/assets/images/tutorial-new-component/example-demos-local.png" />
+	<figcaption class="o-typography-caption">
+        Opening the link output by the develop command, `localhost:8999` in this case, shows us the component demos (html files) and demo assets (js and css files) in the browser.
+	</figcaption>
+</figure>
+
+Clicking `demo.html` to open that demo will show a blank page. In the next section we will update this demo with markup and content for our component.
```

Comment:
```suggestion
Clicking [`demo.html`](http://localhost:8999/demo) to open that demo will show a blank page. In the next section we will update this demo with markup and content for our component.
```
I think having that be a link to the expected demo url would be helpful for the user

Reply:
I'd rather avoid links here mainly as the port number isn't guaranteed  but we also want users to be able to find where the demos are listed as part of this tutorial :)